### PR TITLE
Cypress tests on CI

### PR DESCRIPTION
**Pull request by nienna73**
## What's in this PR:
* Cypress tests run on CI
* create a test DB for Cypress tests
* added fixtures to build the test DB

**Notes:**
I removed the checks for specific CSS values in the Cypress tests. These tests were temperamental and failed in GitHub Actions. Also, checking CSS isn't really the purpose of Cypress.

### CODE REVIEW

File: recvalsite/settings.py [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DEBUG", default=False, cast=bool)
+DEBUG_DJANGO = ""
```
Comment (andrewdotn) [inline recvalsite/settings.py]:
```suggestion
DEBUG_DJANGO = config("DEBUG_DJANGO", default=DEBUG, cast=bool)
```

?

You might also want to rename this to `USE_DJANGO_DEBUG_TOOLBAR`

File: cypress/integration/edit_segment.spec.js [hunk 2]
```diff
@@ -122,7 +115,10 @@ describe("Edit segment", () => {
             })
     })
 
-    it("should load content when clicking Revert", () => {
+    // The following test only works if the entries have a revision history
+    // Since the test-db is made when the tests are run, there is no revision history
+    // to test against
+    it.skip("should load content when clicking Revert", () => {
```
Comment (andrewdotn) [inline cypress/integration/edit_segment.spec.js]: Could you test the revert part after having cypress make a test edit?

**Review by andrewdotn — APPROVED** [review]
I have a few suggestions, but since the tests pass in GitHub Actions, this is great, and I don’t want to hold that up.

File: cypress/integration/edit_segment.spec.js (hunk 2 above)
Comment (nienna73) [reply cypress/integration/edit_segment.spec.js]: Likely, yes. Let me try

Comment (nienna73) [reply cypress/integration/edit_segment.spec.js]: Great suggestion! Putting the revert test after the "save on edit" test solved the problem.

File: recvalsite/settings.py [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DEBUG", default=False, cast=bool)
-DEBUG_DJANGO = ""
+USE_DJANGO_DEBUG_TOOLBAR = config("DEBUG_DJANGO", default=DEBUG, cast=bool)
```
Comment (andrewdotn) [inline recvalsite/settings.py:36]: You’ll want `USE_DJANGO_DEBUG_TOOLBAR` on the right side too, and in the `ci_run_cypress` script.

**Pull request merged**